### PR TITLE
adding unit test cases for add_new_openwrap_partner.py

### PR DESCRIPTION
**Pull request by AvinashKapre**
Increased code coverage of add_new_openwrap_partner.py from  69% to 84%

### CODE REVIEW

File: tests/test_add_new_openwrap_partner.py [hunk 1]
```diff
@@ -118,6 +167,220 @@ def test_missing_adpod_placement_size_1(self, mock_dfp_client):
...
+  @patch('settings.OPENWRAP_CUSTOM_TARGETING',[("a", "IS", (1)), ("b", "IS_NOT", (4))],create=True)
+  def test_wrong_custom_targeting_instance(self,mock_dfp_client):
+    """
+    It throws an exception with a Bad setting.
+    """
+    with self.assertRaises(BadSettingException):
+      tasks.add_new_openwrap_partner.main()
+  
+
+  def test_validate_CSVValues_can_not_negative(self,mock_dfp_client):
```
Comment (Pubmatic-Dhruv-Sonone) [inline tests/test_add_new_openwrap_partner.py:280]: Rename to test_validate_CSVValues_Negative

File: tests/test_add_new_openwrap_partner.py [hunk 2]
```diff
@@ -118,6 +167,220 @@ def test_missing_adpod_placement_size_1(self, mock_dfp_client):
...
+      tasks.add_new_openwrap_partner.validateCSVValues(-1,-2,1,1)
+
+  def test_validate_CSVValues_start_range_more_than_end_range(self,mock_dfp_client):
+    """
+    It throws an exception with Start range can not be more than end range.
+    """
+    with self.assertRaises(BadSettingException):
+      tasks.add_new_openwrap_partner.validateCSVValues(4,1,1,1)
+
+  def test_validate_CSVValues_rate_id_can_1_or_2(self,mock_dfp_client):
```
Comment (Pubmatic-Dhruv-Sonone) [inline tests/test_add_new_openwrap_partner.py:294]: Rename to Invalida Rate ID

File: tests/test_add_new_openwrap_partner.py (hunk 1 above)
Comment (AvinashKapre) [reply tests/test_add_new_openwrap_partner.py:280]: Resolved

File: tests/test_add_new_openwrap_partner.py (hunk 2 above)
Comment (AvinashKapre) [reply tests/test_add_new_openwrap_partner.py:294]: Resolved

**Pull request merged**